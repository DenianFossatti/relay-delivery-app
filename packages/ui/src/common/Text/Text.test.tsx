import { mockTestID, render, wrapper } from '../../tests';

import { Text, TextProps } from '.';

describe(`Text`, () => {
  const props: TextProps = {
    children: 'Welcome',
    testID: 'Text',
  };

  const { findByTestId, toJSON } = render(<Text {...props} />, { wrapper });

  const el_container = mockTestID('Text', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut.children).toEqual([props.children]);
  });

  it('should render snapshot', () => {
    expect(toJSON()).toMatchSnapshot();
  });
});
