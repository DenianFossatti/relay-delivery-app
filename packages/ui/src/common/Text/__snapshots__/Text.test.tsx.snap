// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Text should render snapshot 1`] = `null`;
